### PR TITLE
add thales variant for aes_kwp and regular aes_kwp

### PR DESCRIPTION
**Pull request by william8siew**

### CODE REVIEW

File: cryptoki/src/mechanism/mod.rs [hunk 1]
```diff
@@ -30,11 +30,17 @@ pub struct MechanismType {
 /// mechanism code is 0x80000170 with no parameters 
 pub const CKM_AES_KEY_WRAP_THALES: CK_MECHANISM_TYPE = 0x80000170;
 
+/// Thales has their own defined Key Wrap mechanism which is compatible with PKCS11 standard mechanism AES_KEY_WRAP_PAD
+/// mechanism code is 0x80000171 with no parameters 
+pub const CKM_AES_KEY_WRAP_PAD_THALES: CK_MECHANISM_TYPE = 0x80000170;
```
Comment (stephaniegalang) [inline cryptoki/src/mechanism/mod.rs]: Value is a duplicate of `CKM_AES_KEY_WRAP_THALES`
```suggestion
pub const CKM_AES_KEY_WRAP_PAD_THALES: CK_MECHANISM_TYPE = 0x80000171;
```

File: cryptoki/src/mechanism/mod.rs [hunk 2]
```diff
@@ -30,11 +30,17 @@ pub struct MechanismType {
 /// mechanism code is 0x80000170 with no parameters
```
Comment (stephaniegalang) [inline cryptoki/src/mechanism/mod.rs:30]: Please update this line in the readme: https://github.com/stephaniegalang/rust-cryptoki/blob/ef70a3ed3fc2918a0b4c131fcf2507b14f1199c0/README.md?plain=1#L5

File: cryptoki/src/mechanism/mod.rs (hunk 1 above)
Comment (william8siew) [reply cryptoki/src/mechanism/mod.rs]: Oops thanks for spotting this typo

File: cryptoki/src/mechanism/mod.rs (hunk 2 above)
Comment (william8siew) [reply cryptoki/src/mechanism/mod.rs:30]: done!

**Review by stephaniegalang — APPROVED** [review]

**Pull request merged**